test/config/render-ctmpls.sh

```diff
@@ -4,6 +4,8 @@ env=$1
 vault_token=$2
 working_dir=${3:-$PWD}
 
```

Comment:
@jacmrob what kind of wizardry is happening here? If you compare to the previous commit all I've done is add and echo command, and now things are running through the render-ctmpls.sh script? :|

Reply:
Could there be some kind of weird race condition happening here? my subsequent PR tests are failing, no changes made to this file...

Reply:
#16, #17 are the successful tests that correspond to the 37847e3 and first "jenkins retest" below.